awx/main/models/credential.py

```diff
@@ -81,6 +81,38 @@ def build_safe_env(env):
     return safe_env
 
 
+def check_gateway_for_user_in_organization(user, organization, requesting_user):
+    from ansible_base.resource_registry.tasks.sync import get_resource_server_client
+    from ansible_base.resource_registry.utils.settings import resource_server_defined
+
+    if not resource_server_defined():
+        return False
```

Comment:
In this case, we should use the existing check `actor.has_obj_perm(self.organization, 'view')` so that standalone remains functional.

Reply:
i'm still checking if user belongs to org (locally) in the validate methods, that way it short circuits and we don't even get to this function